Tests/scripts/hook_validations/integration.py

```diff
@@ -87,12 +88,50 @@ def is_default_arguments(self):
                                     .format(arg_name, command_name))
         return self._is_valid
 
+    def is_outputs_for_reputations_commands_valid(self):
+        """Check if a reputation command (domain/email/file/ip/url)
+            has the correct DBotScore outputs according to the context standard
+            https://github.com/demisto/content/blob/master/docs/context_standards/README.MD
+
+        Returns:
+            bool. Whether a reputation command holds valid outputs
+        """
+        context_standard = "https://github.com/demisto/content/blob/master/docs/context_standards/README.MD"
+        commands = self.current_integration.get('script', {}).get('commands', [])
+        for command in commands:
+            command_name = command.get('name')
+            # look for reputations commands
+            if command_name in ['domain', 'email', 'file', 'ip', 'url']:
+                context_outputs_paths = []
+                for output in command.get('outputs', []):
+                    context_outputs_paths.append(output.get('contextPath'))
+
+                # validate DBotScore outputs
+                if 'DBotScore.Indicator' not in context_outputs_paths\
+                        or 'DBotScore.Type' not in context_outputs_paths\
+                        or 'DBotScore.Vendor' not in context_outputs_paths\
+                        or 'DBotScore.Score' not in context_outputs_paths:
+                    self._is_valid = False
+                    print_error("The DBotScore outputs of the reputation command aren't valid. "
+                                "Fix according to context standard {} ".format(context_standard))
+
+                # validate the IOC output
+                if ((command_name == 'domain' and 'Domain.Name' not in context_outputs_paths)
+                        or (command_name == 'file' and 'File.Name' not in context_outputs_paths)
+                        or (command_name == 'ip' and 'IP.Address' not in context_outputs_paths)
+                        or (command_name == 'url' and 'URL.Data' not in context_outputs_paths)):
+                    self._is_valid = False
+                    print_error("The outputs of the {} command aren't valid. "
+                                "Fix according to context standard {} ".format(command_name, context_standard))
+
+        return self._is_valid
+
     def is_valid_subtype(self):
         """Validate that the subtype is python2 or python3."""
         type_ = self.current_integration.get('script', {}).get('type')
         if type_ == 'python':
             subtype = self.current_integration.get('script', {}).get('subtype')
-            if not subtype or subtype not in ['python3', 'python2']:
+            if subtype not in ['python3', 'python2']:
```

Comment:
let's move the subtype options to the constants.py file and use them as a set

Reply:
@yaakovi fixed also for the script.py